src/components/DefaultStyleListDialog.vue

```diff
@@ -163,17 +163,30 @@ watch([() => props.modelValue], async ([newValue]) => {
       .filter(
         (characterInfo) => characterInfo.metas.styles.length > 1
       ) as CharacterInfo[];
-    selectedStyleIndexes.value = Object.fromEntries([
-      ...store.state.userCharacterOrder.map((speakerUuid) => [speakerUuid, 0]),
-      ...store.state.defaultStyleIds.map((defaultStyle) => [
-        defaultStyle.speakerUuid,
-        characterInfosMap.value[
-          defaultStyle.speakerUuid
-        ]?.metas.styles.findIndex(
-          (style) => style.styleId === defaultStyle.defaultStyleId
-        ) ?? 0,
-      ]),
-    ]);
+    selectedStyleIndexes.value = Object.fromEntries(
+      [
+        ...store.state.userCharacterOrder.map(
+          (speakerUuid) => [speakerUuid, 0] as const
+        ),
+        ...store.state.defaultStyleIds.map(
+          (defaultStyle) =>
+            [
+              defaultStyle.speakerUuid,
+              characterInfosMap.value[
+                defaultStyle.speakerUuid
+              ]?.metas.styles.findIndex(
+                (style) => style.styleId === defaultStyle.defaultStyleId
+              ),
+            ] as const
+        ),
+      ].filter(([speakerUuid, styleIndex]) => {
+        // FIXME: もっと根本的な解決方法を考える
```

Comment:
書くなら「起動してないエンジンのpeakerUuidが存在することがあるということを型レベルで気づけるようにする」かなと思いました。
が、そもそもここのコメントに書いてもという感じなので、別途issue化するのが良さそうかも？
（issue化やりますね･･･！）

Reply:
✔️ https://github.com/VOICEVOX/voicevox/issues/1199

Reply:
（このコメントは何を意図しているのか推定するのがだいぶ難しそうなので、消しちゃいますね！）
```suggestion
        // FIXME: もっと根本的な解決方法を考える
```